README.md

```diff
@@ -1 +1,21 @@
 # `web-push-currencies`
+
+## Workflow:
+![image](https://user-images.githubusercontent.com/56842420/193760222-5e43149b-b179-472c-ac7a-f1b8b3fd193b.png)
```

Comment:
Use better quality of the image.
Update these steps: 
2. Separate push service and push manager. Push manager is client-side API that subscribes a user. It sends a request to push service. 
3. Don't split the step
4. should be 'Server triggers push service'